_posts/2023-02-01-fooconf-helsinki.md

```diff
@@ -0,0 +1,91 @@
+
+![paste](/img/2023-fooconf/fooconf2023_2.jpg)
+_T-shirts_
+
+## Talks
+
+The day started with some welcome words by the organizers and after that, there was the first keynote speech of the day, Learning by Tinkering by **Tom Cools**. The speech was about learning and how to keep up with the ever-evolving technologies. The key takeaways of the speech were:
+
+- Limit yourself (do not take on all new tech in the same project)
+- Focus on the concepts that don't decay as fast, like Object Oriented Programming, Functional Programming, TDD, DDD, Design Patterns and Clean code
+- Reflect (check your pet projects in GitLab, check what went wrong)
+- Be conscious about your learning goals (stick to your plan)
+- Alternative approaches (do the same thing many times with another tech/lang, e.g. machine learning)
+- Share what you learned!
+- Do good
+- Be yourself
+
+The speech was really interesting and provided something to think about. The topics covered were common to many developers.
+
+![paste](/img/2023-fooconf/fooconf2023_4.jpg)
+_Learning by Tinkering By Tom Cools_
+
+After the first speech, we took some different tracks. Teemu listened to Cloud-native dev tools by **Grace Jansen** from IBM. It was a good introduction to OSS Java cloud-native technologies like Open Liberty, MicroProfile, Jakarta EE, testing (TestContainers, Microshed) and deploying (Buildpacks.io, Paketo buildpacks, Telepresence). It was a good overview and something to check for future projects as well. Nice demos, tips and tricks. Good to check slides for more information and links.
+
+Markus went to listen to the speech about JavaScript frameworks of tomorrow by **Juho Vepsäläinen** of Aalto University. The speech presented first the past and current trends in frameworks but quickly went on to prospects and the idea of Transitional Web Applications (TWAs). The main goal is to minimize the amount of loaded JavaScript and increase the amount of static content on pages, leading to more efficient web applications. Two approaches of these were introduced: the Island Architecture (11ty/is-land, îles, Capri and Astro -frameworks) and Resumability (Qwik-framework), and demos for Astro and Qwik were shown. Juho also presented his Ph.D. project called Gustwind (with side projects), which is using JSON as the building block for web applications.
+
+Next in line for Teemu was a very nice speech about how a Legacy app went to serverless bar by **Sébastien Blanc** from Aiven. It was a humorous story featuring a 13 years old legacy app called Bob which wanted to go cloud and serverless. The story went through very humorously including funny characters like Join the table, Maria, Kafka river etc and ended up in an Aiven product demo, about serverless, Kafka, Kubernetes and autoscaling.
```

Comment:
"humorous" used twice very close to eachother makes for a stuttering effect in the flow. Maybe sth like

"The story included funny characters like..." which would make for a more natural flow without repetition.

Reply:
Fixed, thanks!